website/content/partials/update-primary-known-issue.mdx

```diff
@@ -2,22 +2,15 @@
 
 #### Affected versions
 
-- All current versions of Vault
-
-<Tip title="We are actively working on the underlying issue">
-
-Look for **Fix a race condition with update-primary that could result in data
-loss after a DR failover.** in a future changelog for the resolution.
-
-</Tip>
+- All versions of Vault prior to 1.14, 1.13.4, 1.12.8, and 1.11.12.
```

Comment:
I'm not sure what we're trying to say here.  Are we trying to say that it's all minor versions within a major release that are less than the values listed? If so, it would be clearer if we said something like:

```
- Vault 1.11.0 – 1.11.11
- Vault 1.12.0 – 1.12.7
- Vault 1.13.0 – 1.13.3
```

If we're trying to say that it's all versions *except* the ones listed, we should just say that:

```
- All version of Vault **except** 1.14.x, 1.13.4, 1.12.8, and 1.11.12
```